src/mapnik_vector_tile.cpp

```diff
@@ -4418,8 +4418,10 @@ v8::Local<v8::Value> VectorTile::_getDataSync(Nan::NAN_METHOD_ARGS_TYPE info)
                 if (release)
                 {
                     std::unique_ptr<std::string> out = d->tile_->release_buffer();
-                    return scope.Escape(Nan::NewBuffer(&((*out)[0]),
-                                                       out->size(),
+                    char * data = &((*out)[0]);
```

Comment:
Could this be re-written as: `out->data()` or `out->c_str()` for a bit better readability?

Reply:
Sadly no, because those return `const char *` and NewBuffer takes `char *`.